manage_proj_edit_page.php

```diff
@@ -492,11 +499,25 @@
 			<input type="submit" name="copy_to" class="btn btn-sm btn-primary btn-white btn-round" value="<?php echo lang_get( 'copy_versions_to' ) ?>" />
 		</fieldset>
 	</form>
-	</div>
+				</div>
+				<div class="pull-right">
+    <form id="manage-project-version-filter" method="post" action="manage_proj_edit_page.php?project_id=<?php echo $f_project_id ?>" class="form-inline">
```

Comment:
It would be more user-friendly if you jumped back to the Versions section of the page after submitting the form.